<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox_main">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QScrollArea" name="scrollArea_display">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>858</width>
              <height>645</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QTextEdit" name="textEdit_display"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_allStats">
             <item>
              <widget class="QPushButton" name="button_refresh">
               <property name="text">
                <string>Re-Calculate Stats</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_refreshDisplay">
               <property name="text">
                <string>Show Full Datalog</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_raiders">
             <item>
              <widget class="QPushButton" name="button_cell">
               <property name="text">
                <string>Add Cell</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_frieza">
               <property name="text">
                <string>Add Frieza</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_buu">
               <property name="text">
                <string>Add Buu</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_vegeta">
               <property name="text">
                <string>Add Vegeta</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_ginyu">
               <property name="text">
                <string>Add Ginyu</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_pickStats">
             <item>
              <widget class="QPushButton" name="button_submit">
               <property name="text">
                <string>Submit Selection</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_clear">
               <property name="text">
                <string>Clear Selection</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
